src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -76,11 +80,12 @@ public class NewAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         [ValidateNotNullOrEmpty]
         public AllowReadOnlyFailoverToPrimary AllowReadOnlyFailoverToPrimary { get; set; }
 
-        /// <summary>
-        /// Gets or sets the tag associated with the Azure SQL Database Failover Group
-        /// </summary>
+        /// <summary> 
+        /// Gets or sets the tags associated with the Azure SQL Database Failover Group 
+        /// </summary> 
         [Parameter(Mandatory = false,
-            HelpMessage = "The tag to associate with the Azure SQL Database Failover Group")]
+            HelpMessage = "The tags to associate with the Azure SQL Database Failover Group")]
+        [Obsolete("This parameter is only for backwards compatibility; User should not need to pass in this parameter.")]
```

Comment:
Yes, we changed our design and will no longer support tag in the future for failover group